src/docs/concepts-transactions/default-negotiation-process/index.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Default negotiation process
+slug: default-negotiation-process
+updated: 2025-09-16
+category: concepts-transaction-process
+ingress:
+  Sharetribe has a default negotiation transaction process that enables
+  users to negotiate a price for the transaction. This article explains
+  how the negotiation process works.
+published: true
+---
+
+# Negotiation process overview
+
+One of the default Sharetribe transaction processes includes price
+negotiation dynamics. The same process enables both forward and reverse
+transaction flows.
+
+- **Forward transaction flow**
```

Comment:
Suggestion: clarify that this is the "normal" transaction flow

Reply:
Also, these could be level 3 subheadings instead of bullet points if it makes sense structurally.